sale_product_pack/models/sale_order_line.py

```diff
@@ -88,7 +90,18 @@ def check_pack_line_modify(self):
         """ Do not let to edit a sale order line if this one belongs to pack
         """
         if self._origin.pack_parent_line_id and \
-           not self._origin.pack_parent_line_id.product_id.pack_modifiable:
+           not self._origin.pack_modifiable:
             raise UserError(_(
                 'You can not change this line because is part of a pack'
                 ' included in this order'))
+
+    @api.multi
+    def action_openParentPackProductView(self):
```

Comment:
Please use always snake case for method variables:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#342variable-names